internal/charm/repository/charmhub_test.go

```diff
@@ -1180,31 +1180,6 @@ func (*selectNextBaseSuite) TestSelectNextBaseWithValidBases(c *gc.C) {
 	}})
 }
 
-func (*selectNextBaseSuite) TestSelectNextBaseWithValidBasesWithSeries(c *gc.C) {
-	repo := new(CharmHubRepository)
-	platform, err := repo.selectNextBases([]transport.Base{{
-		Architecture: "amd64",
-		Name:         "ubuntu",
-		Channel:      "focal",
-	}, {
-		Architecture: "amd64",
-		Name:         "ubuntu",
-		Channel:      "20.04",
-	}}, corecharm.Origin{
-		Platform: corecharm.Platform{
-			Architecture: "amd64",
-			OS:           "ubuntu",
-			Channel:      "20.04",
-		},
-	})
-	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(platform, gc.DeepEquals, []corecharm.Platform{{
-		Architecture: "amd64",
-		OS:           "ubuntu",
-		Channel:      "20.04",
-	}})
-}
```

Comment:
I don't believe this test is valid

I don't think charmhub can return a series in `.results[].error.extra.default-bases[].channel`

@hmlanigan what do you think?

Reply:
Originally charms did do this with charm-hub. Slowly it changed from series name to a os channel we know today.  It depends on how the charm was written and what charm craft allows now.  It could be historical at this time, but I have no guarantees it is either. 

They may have fixed this on the charm hub server side, run:
```
$ curl -s https://api.snapcraft.io/v2/charms/refresh -H 'Content-type: application/json' -d '{"context":[],"actions":[{"action":"install","instance-key":"testing","name":"ubuntu","revision":14,"base":null}],"fields":["bases","config-yaml","download","id","license","metadata-yaml","name","publisher","resources","revision","summary","type","version"]}' | jq .
```

This charm was migrated in from charmstore and has:
```
        "bases": [
          {
            "architecture": "amd64",
            "channel": "14.04",
            "name": "ubuntu"
          }
```